ca/certificate-authority.go

```diff
@@ -62,13 +62,15 @@ type CertificateAuthorityImpl struct {
 // using CFSSL's authenticated signature scheme.  A CA created in this way
 // issues for a single profile on the remote signer, which is indicated
 // by name in this constructor.
-func NewCertificateAuthorityImpl(cadb core.CertificateAuthorityDatabase, config Config) (ca *CertificateAuthorityImpl, err error) {
+func NewCertificateAuthorityImpl(cadb core.CertificateAuthorityDatabase, config Config) (*CertificateAuthorityImpl, error) {
+	var ca *CertificateAuthorityImpl
```

Comment:
I *think* it's okay, even if you're using clothed returns, to declare the variables in the return statement of the function. You would just name them explicitly in the return. But I'm not 100% on go standard style, so feel free to correct me.